src/scenes/ApplicationRouter/portal/ApplicationPage.tsx

```diff
@@ -36,6 +40,11 @@ const ApplicationPage = ({ children }: Prop) => {
         title={strings.APPLICATION}
         rightComponent={<Button label={strings.EXIT_APPLICATION} onClick={goToHome} priority={'ghost'} />}
       />
+      <Grid container spacing={theme.spacing(1)}>
+        <Grid item xs style={{ flexGrow: 1, padding: `${theme.spacing(2)} ${theme.spacing(4)}` }}>
+          {crumbs && <BreadCrumbs crumbs={crumbs} hierarchical={hierarchicalCrumbs ?? true} />}
+        </Grid>
+      </Grid>
```

Comment:
I copied this grid structure from `ModuleDetailsCard` so the structure would be similar, but perhaps all that's needed is a `Box` with the correct margins/padding?

Reply:
Sorry I probably should have caught this earlier. It may be easier to use the `Page` component instead of TFMain.

I took this from the deliverable view but the title + breadcrumb style actually fits better!

![Screenshot 2024-07-24 at 4.30.35 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/NOxkRPuhAfT4F7nmdXtG/1a7b2af5-f633-4236-98ff-132a6236bc22.png)

Reply:
Ah and to avoid confusion, this screenshot was from Terraware staging on the DeliverableListView. It has the breadcrumbs on top of title too! It uses the `Page` component which is why I suggested it.

Reply:
I didn't notice these comments until just before I was about to merge, but I updated to use the `Page` component ✅

Reply:
Sorry about the late comments! I will take another quick review then once Vercel is done!